src/metpy/plots/patheffects.py

```diff
@@ -0,0 +1,245 @@
+        """Return the proper symbol to draw; alternatives between scallop and pip/triangle."""
+        if self._symbol_cycle is None:
+            self._symbol_cycle = itertools.cycle([WarmFront.symbol, ColdFront.symbol])
+        return next(self._symbol_cycle)
+
+
+@exporter.export
+class StationaryFront(Front):
+    """Draw a stationary front as alternating cold and warm front segments."""
+
+    symbol = WarmFront.symbol
+    symbol2 = ColdFront.symbol.transformed(mtransforms.Affine2D().scale(1, -1))
+
+    def __init__(self, colors=None, **kwargs):
+        if colors is None:
+            colors = ['red', 'blue']
+        self._colors = list(map(mcolors.to_rgba, colors))
+        super().__init__(color=self._colors[0], **kwargs)
+
+    def _get_path_segment_ends(self, segment_offsets, renderer):
+        # Calculate increment of path length occupied by each marker drawn
+        inc = self._step_size(renderer)
+
+        # Find out how many markers that will accommodate, as well as remainder space
+        num, leftover = divmod(segment_offsets[-1], inc)
+
+        # Find the offset for each path segment end. We center along
+        # the entire path by adding half of the remainder.
+        path_offsets = np.arange(1, num + 1) * inc + leftover / 2.
+
+        # Find the location of these offsets within the total offset within each
+        # path segment; subtracting 1 gives us the left point of the path rather
+        # than the last.
+        return np.searchsorted(segment_offsets, path_offsets)
+
+    def draw_path(self, renderer, gc, path, affine, rgbFace=None):  # noqa: N803
+        """Draw the given path."""
+        gcs = [self._override_gc(renderer, gc, foreground=color) for color in self._colors]
+        self._gc_cycle = itertools.cycle(gcs)
+        self._symbol_cycle = itertools.cycle([self.symbol, self.symbol2])
+        self._color_cycle = itertools.cycle(self._colors)
+
+        # Get the information we need for drawing along the path
+        starts, offsets, angles = self._process_path(path, affine)
+
+        # Figure out what segments the markers should be drawn upon and how
+        # far within that segment the markers will appear.
+        segment_indices, marker_offsets = self._get_marker_locations(offsets, renderer)
+        end_path_inds = self._get_path_segment_ends(offsets, renderer)
+        start_path_inds = np.concatenate([[0], end_path_inds[:-1]])
+
+        # Need to account for the line width in order to properly draw symbols at line edge
+        line_shift = renderer.points_to_pixels(gc.get_linewidth()) / 2
+
+        # Loop over all the markers to draw
+        for ind, start_path, end_path, marker_offset in zip(segment_indices, start_path_inds,
+                                                            end_path_inds, marker_offsets):
+            sym_trans = self._get_symbol_transform(renderer, marker_offset, line_shift,
+                                                   angles[ind], starts[ind])
```

Comment:
<img width="484" alt="image" src="https://user-images.githubusercontent.com/24628426/230660361-2e135413-eedf-4d79-af91-8e90c45ae619.png">

The current result of `StationaryFront(flip=True)` on my machine. I haven't dove too deep yet, but I assumed this was where I'd start.

Reply:
🤔 Well I could have sworn that worked before...which means I probably broke something with a "clean up" refactor

Reply:
Ok, fixed and have now included `flip` in this test.